build-release.sh

```diff
@@ -105,17 +105,7 @@ if [ ! -e "$RECIPE" ]; then
   git commit -m "Added recipe for ${VERSION}"
 else
   PREV_VERSION=$(awk -F / '/^  url/{print $NF}' "$RECIPE" | gsed 's/^.\+-\([0-9].\+\)\.tar.\+/\1/')
-  if [ "$PREV_VERSION" = "$VERSION" ]; then
-    # recipe already exists and has the correct HHVM version (presumably because
-    # a package for at least one other OS version was already built); no need to
-    # bump-formula, but we still need to delete any existing bottle references
-    # for the current OS version (from any potential previous bad/outdated
-    # builds)
-    source os-metadata.inc.sh
-    gsed -i "/sha256.\+ => :$OS_CODENAME/d" "${RECIPE}"
```

Comment:
One thing I'm not sure of: Do we also need to keep this here (in addition to below)? i.e. will `brew install`/`brew bottle` fail if it sees an existing bottle in the formula file?

Reply:
Say you build Mojave, but Catalina fails. You retry catalina. The formula now automatically gets build version 2 ... but the mojave bottle is for build version one, so is no longer installable as it's outdated. Rebuild that, and it's version 3, and catalina is now outdated.

Reply:
actually no, not in addition to below